core/src/main/java/io/questdb/std/datetime/microtime/TimestampFormatUtils.java

```diff
@@ -289,30 +289,29 @@ public static long compute(
             throw NumericException.INSTANCE;
         }
 
-        long datetime;
-        if (week == 0) {
-            datetime = Timestamps.yearMicros(year, leap)
-                    + Timestamps.monthOfYearMicros(month, leap)
-                    + (day - 1) * Timestamps.DAY_MICROS
-                    + hour * Timestamps.HOUR_MICROS
-                    + minute * Timestamps.MINUTE_MICROS
-                    + second * Timestamps.SECOND_MICROS
-                    + (long) millis * Timestamps.MILLI_MICROS
-                    + micros;
-        } else {
-            long yearMicros = Timestamps.yearMicros(year, leap);
-            // 4 Jan of year Y
-            // correction formula is taken from https://en.wikipedia.org/wiki/ISO_week_date
-            final long correction = Timestamps.getDayOfWeek(yearMicros + Timestamps.monthOfYearMicros(1, leap) + 4 * Timestamps.DAY_MICROS) + 3;
-            datetime = yearMicros
-                    + (week * 7L + day - correction) * Timestamps.DAY_MICROS
-                    + hour * Timestamps.HOUR_MICROS
-                    + minute * Timestamps.MINUTE_MICROS
-                    + second * Timestamps.SECOND_MICROS
-                    + (long) millis * Timestamps.MILLI_MICROS
-                    + micros;
+        if (week < 0 || week > 53) {
```

Comment:
Week can never be 0, that is why I did not include it in the ISO week calculations. Also, due to ISO weeks affecting parsing in a major way, I wanted there to be a default way to avoid parsing with ISO weeks. Since weeks is 0 by default, it will not be parsed as an ISO week unless explicitly changed.

Reply:
If it's not what causing for `2020W00` to parse than perhaps it can be left as `<`.
See the other comment with test scenarios.